rust/operator-binary/Cargo.toml

```diff
@@ -1,40 +1,37 @@
 [package]
-authors = ["Lars Francke <lars.francke@stackable.de>"]
+authors = ["Stackable GmbH <info@stackable.de>"]
+build = "build.rs"
 description = "Stackable Operator for Trino"
-edition = "2018"
+edition = "2021"
 license = "OSL-3.0"
-name = "stackable-trino-operator-binary"
+name = "stackable-trino-operator"
 repository = "https://github.com/stackabletech/trino-operator"
 version = "0.3.0-nightly"
-build = "build.rs"
 
 [dependencies]
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.4.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.8.0" }
 stackable-trino-crd = { path = "../crd" }
-stackable-trino-operator = { path = "../operator" }
 
-clap = "2.33"
-tokio = { version = "1.10", features = ["macros", "rt-multi-thread"] }
+anyhow = "1.0"
+clap = "3.0"
+futures = { version = "0.3", features = ["compat"] }
+pin-project = "1.0"
+semver = "1.0"
+serde = "1.0"
+serde_json = "1.0"
+serde_yaml = "0.8"
+snafu = "0.6"
+structopt = "0.3"
+strum = "0.23"
+strum_macros = "0.23"
+tokio = { version = "1.12", features = ["full"] }
 tracing = "0.1"
 
 [build-dependencies]
 built = { version =  "0.5", features = ["chrono", "git2"] }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.4.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.8.0" }
 stackable-trino-crd = { path = "../crd" }
 
-[package.metadata.deb]
-name = "stackable-trino-operator"
-maintainer-scripts = "../../packaging/debian/"
-systemd-units = { enable = false, unit-name = "stackable-trino-operator" }
-assets = [
-    ["../../target/release/stackable-trino-operator", "opt/stackable/trino-operator/", "755"],
-    ["../../deploy/crd/trinocluster.crd.yaml", "etc/stackable/trino-operator/crd/", "644"],
-    ["../../deploy/crd/restart.crd.yaml", "etc/stackable/trino-operator/crd/", "644"],
-    ["../../deploy/crd/start.crd.yaml", "etc/stackable/trino-operator/crd/", "644"],
-    ["../../deploy/crd/stop.crd.yaml", "etc/stackable/trino-operator/crd/", "644"],
-    ["../../deploy/config-spec/properties.yaml", "etc/stackable/trino-operator/config-spec/", "644"],
-]
-
 [[bin]]
 name = "stackable-trino-operator"
-path = "src/stackable-trino-operator.rs"
+path = "src/main.rs"
```

Comment:
No need for this section at all then, since it's implied for `src/main.rs`.

Reply:
removed